.github/workflows/build-deb-pkg.sh

```diff
@@ -0,0 +1,32 @@
+PKG_ROOT=/tmp/avalanche
+DEBIAN_BASE_DIR=$PKG_ROOT/debian
+AVALANCHE_BUILD_BIN_DIR=$DEBIAN_BASE_DIR/usr/local/bin
+AVALANCHE_LIB_DIR=$DEBIAN_BASE_DIR/usr/local/lib/avalanche
+TEMPLATE=.github/workflows/debian/template 
+DEBIAN_CONF=$DEBIAN_BASE_DIR/DEBIAN
+
+mkdir -p $DEBIAN_BASE_DIR
+mkdir -p $DEBIAN_CONF
+mkdir -p $AVALANCHE_BUILD_BIN_DIR
+mkdir -p $AVALANCHE_LIB_DIR
+
+OK=`cp ./build/avalanche $AVALANCHE_BUILD_BIN_DIR`
```

Comment:
The binary formerly known as `gecko`, and `avalanche` is now called `avalanchego`. This (and probably other parts) will need updating to reflect that.